modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -157,6 +164,41 @@ public List<AccountGroupRel> getAccountGroupRels(
 			accountGroupId, _classNameLocalService.getClassNameId(className));
 	}
 
+	@Override
+	public List<AccountGroupRel> getAccountGroupRels(
+		long cpDefinitionId, String keywords, int start, int end) {
```

Comment:
This is wrong already. Do not use commerce related naming schemes in non-COMMERCE modules. It needs to be `long classPK`.

Reply:
Also, you should also match `className` in the query.

Reply:
Expected is:
```
String className, long classPK, String keywords, int start, int end) {
```